plugins/inputs/ipvs/ipvs.go

```diff
@@ -81,6 +110,17 @@ func serviceTags(s *ipvs.Service) map[string]string {
 	return ret
 }
 
+// helper: given a Destination, return tags that identify it
+func destinationTags(d *ipvs.Destination) map[string]string {
+	return map[string]string{
+		"address":        d.Address.String(),
+		"port":           fmt.Sprintf("%d", d.Port),
+		"address_family": addressFamilyToString(d.AddressFamily),
+		"active_conns":   fmt.Sprintf("%d", d.ActiveConnections),
+		"inactive_conns": fmt.Sprintf("%d", d.InactiveConnections),
```

Comment:
These, active/inactive connections, won't make good tags because they will change frequently, which will cause high series cardinality, and as tags you won't be able to do mathematical operations.  We should change them to fields.

Reply:
Good catch. The intention was to make them fields, but I may have had a brain-fart at the time.